File: RB101/exercises/easy_4/short_long_short.rb
def short_long_short(input_1, input_2)
  if input_1.size > input_2.size
    puts input_2 + input_1 + input_2
  else 
    puts input_1 + input_2 + input_1
  end
end

puts short_long_short('abc', 'defgh')
puts short_long_short('abcde', 'fgh')

File: JS210/exersies/easy1/short_long_short.js
function shortLongShort(string1, string2) {
  let stringLength1 = string1.length;
  let stringLength2 = string2.length;

  if (stringLength1 < stringLength2) {
    console.log(string1 + string2 + string1)
  } else {
    console.log(string2 + string1 + string2)
  }
}



shortLongShort('abc', 'defgh');    // "abcdefghabc"
shortLongShort('abcde', 'fgh');    // "fghabcdefgh"
shortLongShort('', 'xyz');         // "xyz"